docs/DeveloperGuide.adoc

```diff
@@ -812,18 +812,20 @@ These instructions only provide a starting point for testers to work on; testers
 .. Copy the jar file to the folder you would like to use as a home address for Delino application
    Expected: Shows the GUI of the Delino App. The window size may not be optimum
 
-=== Adding an order
+=== Inserting an order
 
-. Add a minimum of 2 orders
+. Insert a minimum of 2 orders
 
-.. Add command format: `add` `tid/TRANSACTION_ID` `n/NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `[c/COMMENTS_BY_CUSTOMER]` `[type/TYPE_OF_ITEM]`
+.. Insert command format: `insert` `tid/TRANSACTION_ID` `n/CUSTOMER_NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `[c/COMMENTS_BY_CUSTOMER]` `[type/TYPE_OF_ITEM]`
 
-.. Test case: `add` `tid/9876543210` `n/John Doe` `a/Blk 572 Hougang st 51 #10-33 S530572` `p/98766789` `ts/2020-02-20 1300` `w/Yishun` `pm/Cash` +
-   Expected: An order with the above details will be added to the list and displayed on the GUI
-.. Test case: `add` `tid/1023456789` `n/Amos Cheong` `a/Blk 572 Hougang st 51 #11-37 S530572` `p/9001 0019` `ts/2020-03-10 1650` `w/Marsiling` `pm/Nets` `c/Leave it at the riser` `type/glass`
-   Expected: The order is added to the list, including the item type and the order comment
+.. Test case: `insert` `tid/9876543210` `n/John Doe` `a/Blk 572 Hougang st 51 #10-33 S530572` `p/98766789` `ts/2020-02-20 1300` `w/Yishun` `pm/Cash` +
+   Expected: Inserts an order with the above details to the list and displayed on the GUI
+.. Test case: `insert` `tid/1023456789` `n/Amos Cheong` `a/Blk 572 Hougang st 51 #11-37 S530572` `p/9001 0019` `ts/2020-03-10 1650` `w/Marsiling` `pm/Nets` `c/Leave it at the riser` `type/glass`
+   Expected: Inserts the order to the list, including the item type and the order comment
 .. Test case: Invalid Syntax
-   Expected: No order is added. Error details shown in the response message. A help message displayed for user to add accordingly. Status bar remain unchanged
+   Expected: No order is added. Error details shown in the response message. A help message displayed for user to insert accordingly. Status bar remain unchanged
+.. Test case: Insert order with existing Transaction ID in list
+   Expected: An error will occur and a message will be displayed, stating that order with duplicate ID cannot be inserted into the list
```

Comment:
There should be a newline for 'Expected' section in DeveloperGuide's G2: Inserting an order 
(Appendix G: Instructions for Manual Testing)
![image](https://user-images.githubusercontent.com/38811217/76157013-f0daec80-613d-11ea-94f7-80491c159c00.png)